packages/neuron-wallet/src/utils/export-history.ts

```diff
@@ -43,7 +43,9 @@ const exportHistory = async ({
     `${headers.map(label => t(`export-transactions.column.${label}`))}\n`
   )
 
-  const addresses = AddressService.allAddressesByWalletId(walletID).map(addr => addr.address)
+  const allAddresses = await AddressService.allAddressesByWalletId(walletID)
```

Comment:
`allAddresses` and `addresses` are confusing

Reply:
Please see the latest commits